src/main/flight/mixer_tricopter.h

```diff
@@ -73,4 +73,82 @@ int16_t triGetMotorCorrection(uint8_t motorIndex);
  */
 _Bool triEnableServoUnarmed(void);
 
+#ifdef USE_SERVOS
```

Comment:
I have used two different ways in the past with unit testing frameworks that don't support accessing statics or including the SUT .c-file in the test file:

1. split them to a separate *_internal.h header file
2. declare them in the public header file but enclose as follows:
```
#ifdef MIXER_TRICOPTER_INTERNALS
declare stuff
#endif

```
Then when including the header file in mixer_tricopter.c or in the test file you would have to
```
#define MIXER_TRICOPTER_INTERNALS
#include "mixer_tricopter.h"
```
This guards against unintentional external usage.